### PR TITLE
Update meshroom_photogrammetry to meshroom_batch in documentation & wiki

### PR DESCRIPTION
**Pull request by ownrepo1s**
Update documentation to align with binary name change.

https://github.com/alicevision/meshroom/commit/37ffd7f990f4a1cf6de67df0fad1dbc4c89b74a7

@fabiencastan next time consider changing the documentation as well, it was quite a search and took some time to find what happened here.

I'm unsure how to edit the wikis: 
https://github.com/alicevision/AliceVision/wiki/CLI-Parameters
https://github.com/alicevision/meshroom/wiki/2019.2-release-notes

Any pointers would be appreciated

### CODE REVIEW

**Comment by natowi** [comment]
Thank you. I need to bring the documentation up to date, since there were many changes.
The Github wiki will eventually be merged with the documentation. I keep the 2019.2 release notes, as they contain valuable information I want to add to the docs eventually.

File: source/feature-documentation/cmd/photogrammetry.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-``meshroom_photogrammetry``
+``meshroom_batch formerly meshroom_photogrammetry``
```
Comment (fabiencastan) [inline source/feature-documentation/cmd/photogrammetry.rst]:
```suggestion
``meshroom_batch (formerly meshroom_photogrammetry)``
```

File: source/first-steps/install/getting-meshroom/index.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ Meshroom binaries for Windows platform and Linux platform
 can be downloaded from `here <https://alicevision.github.io/#meshroom>`_.
 
 Prebuilt binaries on this page are all-in-one packages including AliceVision and all required resources.
-The pre-built binaries also contain the ``meshroom_compute`` and ``meshroom_photogrammetry`` to run and create pipelines from the command line.
+The pre-built binaries also contain the ``meshroom_compute`` and ``meshroom_batch``, formerly known as ``meshroom_photogrammetry`` to run and create pipelines from the command line.
```
Comment (fabiencastan) [inline source/first-steps/install/getting-meshroom/index.rst]:
```suggestion
The pre-built binaries also contain the ``meshroom_compute`` and ``meshroom_batch`` (formerly known as ``meshroom_photogrammetry``) to run and create pipelines from the command line.
```

**Comment by simogasp** [comment]
Can we merge?

**Review by natowi — APPROVED** [review]

**Comment by ownrepo1s** [comment]
@fabiencastan  Only those with write access to this repository can merge pull requests.

**Pull request merged**